### ISSUE TITLE
FCN Skip Connections

### ISSUE THREAD

**Issue by xternalz** [warmspringwinds/pytorch-segmentation-detection#3]
Hi, FCN-8s/16s (regardless of the base model being VGG/ResNet) should have skip connections for aggregating the features from pooling layers. But, I can't seem to find these in your model definitions.

**Comment by warmspringwinds** [comment]
@xternalz you are right. FCN-16s, and FCN-8s will be implemented in a future, or you can contribute if you don't want to wait. Although we have other models implemented based on dilated convolutions.
